### ISSUE TITLE
/api/v0/blueprints/list fails: ReferenceError: cockpit is not defined

### ISSUE THREAD

**Issue by martinpitt** [weldr/welder-web#617]
### Description of problem

In OSCI I've seen several [test failures](https://jenkins.osci.redhat.com/job/rhel-8-build-pipeline/10957/) like this:
```
75 passing (746.20s)
1 failing

1) lorax-composer api sanity test /api/v0/blueprints/list:
ReferenceError: cockpit is not defined
running firefox
Error: An error occurred while executing user supplied JavaScript.
    at executeAsync(<Function>, "/api/v0/blueprints/list", [object Object]) - index.js:312:3
```

**Comment by henrywang** [comment]
@martinpitt , This issue was fixed by [PR#610](https://github.com/weldr/welder-web/pull/610) already.

**Comment by martinpitt** [comment]
I see, thanks! That just didn't make it into a release yet.
